examples/quantization/rep_qat.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import os
+import sys
+
+CURRENT_PATH = os.path.abspath(os.path.dirname(__file__))
+
+sys.path.insert(1, os.path.join(CURRENT_PATH, '../../'))
+
+import functools
+from distutils.version import LooseVersion, StrictVersion
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torch.quantization as torch_q
+
+from examples.models.cifar10.mobilenet import DEFAULT_STATE_DICT, Mobilenet
+from tinynn.converter import TFLiteConverter
+from tinynn.graph.quantization.quantizer import QATQuantizer, PostQuantizer, load_processed_ptq_rules
+from tinynn.graph.tracer import model_tracer, trace
+from tinynn.util.cifar10 import get_dataloader, train_one_epoch, validate, calibrate
+from tinynn.util.train_util import DLContext, get_device, train
+
+from tinynn.util.cross_layer_equal import weight_euqal_
```

Comment:
It is more reasonable to move `cross_layer_equal` to `tinynn/graph/quantization directory`